docs/security.md

```diff
@@ -63,10 +65,42 @@ We use a number of static analysis tools to identify security risks in developme
 
 All development laptops have encrypted hard drives.
 
-## Bug reports
+## How to report a security vulnerability
+
+If you think that you have found a security issue, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to high quality reports within 1 business day to acknowledge that we received them and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to or issue bounties for low quality or spammy reports (e.g. those produced by automated tooling).
+
+We provide monetary rewards, up to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.
+
+We will send payment to a valid PayPal account after the issue is confirmed fixed or 90 days from the original report, whichever happens first. We will ask you for the name and country associated with your PayPal account.
+
+We will not issue a reward if any of the following apply:
+
+1. You engage in disruptive behavior on sourcegraph.com itself (e.g. spamming our system with requests, fake accounts, denial of service). Sourcegraph is [open source software](https://github.com/sourcegraph/sourcegraph), so you can [install a copy yourself](https://docs.sourcegraph.com/#quickstart-guide) and test against that instead.
+1. You publically disclose a vulnerability before we have fixed it.
+1. You spam us with duplicate and/or low quality vulnerability reports (e.g. copy/pasting generic issues from automatic scanning tools).
+1. You are currently a teammate at Sourcegraph (e.g. employee, contractor, intern).
+
+We define multipliers for each of the factors for the reward amount and apply these multipliers against our maximum bounty ($10,000 USD). For example, a vulnerability that impacts a small number of customers (0.5x), is difficult to exploit (0.5x), and would cause a lot of harm (1x), might be awarded $2,500 USD (i.e. `$10,000 * 0.5 * 0.5 * 1.0 = $2,500`).
+
+These are only guidelines and the final award amount for any issue is at the discretion of Sourcegraph.
+
+
+| Vulnerable customers* | Reward multiplier |
+|----------------------|-------------------|
+| A majority (> 50%) of customers | 1x |
+| A small number of customers | 0.5x |
+| No customers | 0.01x |
+
+\* A **customer** is defined as a paid user of Sourcegraph. Please note that there are currently no paying customers on sourcegraph.com because sourcegraph.com only hosts public code and is free to use. As such, an issue that impacts all **users** on sourcegraph.com doesn't necessarily impact any **customers**.
```

Comment:
This indicates that someone reporting a major vulnerability in Sourcegraph.com that leaks, say, GitHub OAuth tokens, would receive $10,000 * 0.01x or just $100, which I assume can't be correct?

Reply:
Yeah, multipliers are hard. I removed them.